Logger.cs

```diff
@@ -1,324 +1,174 @@
+            {
+                Log.WriteLine($"LOG DO DIA: {DateTime.Now.ToShortDateString()}");
+            }
         }
-        void WriteTxt()
+
+        private void ReadTxt()
         {
-            StreamWriter writer = new StreamWriter(myDir + nameOfFile);
-            writer.WriteLine(pastMessage);
-            writer.Write(DateTime.Now.ToLongTimeString() + ": " + message);
-            writer.Close();
-            writer.Dispose();
-            writer = null;
+            using (var writer = new StreamReader(Path.Combine(dir, nameOfFile)))
+            {
+                pastMessage = writer.ReadToEnd();
+            }
         }
-        void WriteTxt_NewLine()
+
+        private void WriteTxt()
         {
-            StreamWriter writer = new StreamWriter(myDir + nameOfFile);
-            writer.WriteLine(pastMessage);
-            writer.WriteLine();
-            writer.Write(DateTime.Now.ToLongTimeString() + ": " + message);
-            writer.Close();
-            writer.Dispose();
-            writer = null;
+            using (var writer = new StreamWriter(Path.Combine(dir, nameOfFile)))
+            {
+                writer.WriteLine(pastMessage);
+                writer.Write($"{DateTime.Now.ToLongTimeString()} : {message}");
+            }
         }
-        void JumpLine()
+
+        private void WriteTxtOnNewLine()
         {
-            StreamWriter writer = new StreamWriter(myDir + nameOfFile);
-            writer.WriteLine(pastMessage);
-            writer.WriteLine();
-            writer.Close();
-            writer.Dispose();
-            writer = null;
+            using (var writer = new StreamWriter(Path.Combine(dir, nameOfFile)))
+            {
+                writer.WriteLine(pastMessage);
+                writer.WriteLine();
+                writer.Write($"{DateTime.Now.ToLongTimeString()} : {message}");
+            }
         }
-        void Clear()
+
+        private void Clear()
         {
-            myDir = Path.GetDirectoryName(System.AppDomain.CurrentDomain.BaseDirectory.ToString());
-            nameOfFile = String.Empty;
-            message = String.Empty;
+            dir         = Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory);
+            nameOfFile  = String.Empty;
+            message     = String.Empty;
             pastMessage = String.Empty;
         }
-        void ErrorWritter()
+
+        private void ErrorWriter()
         {
-            StreamWriter writer = new StreamWriter(Path.GetDirectoryName(System.AppDomain.CurrentDomain.BaseDirectory.ToString()) + "UnknownError.txt");
-            writer.Write(DateTime.Now.ToLongTimeString() + ": " + "Verify your directory.");
-            writer.Close();
-            writer.Dispose();
-            writer = null;
+            using (var writer = new StreamWriter(Path.GetDirectoryName(System.AppDomain.CurrentDomain.BaseDirectory.ToString()) + "UnknownError.txt"))

```

Comment:
The argument to Path.GetDirectoryName can be cleaned up a little.
- System prefix can be removed
- Use Path.Combine with BaseDirectory and "UnknownError.txt"